browser/ui/webui/brave_new_tab_message_handler.cc

```diff
@@ -282,6 +285,9 @@ void BraveNewTabMessageHandler::OnJavascriptAllowed() {
   pref_change_registrar_.Add(kNewTabPageShowGemini,
     base::Bind(&BraveNewTabMessageHandler::OnPreferencesChanged,
     base::Unretained(this)));
+  pref_change_registrar_.Add(kNewTabPageShowBitcoinDotCom,
```

Comment:
should be within `#if BUILDFLAG(BITCOIN_DOT_COM_ENABLED)`

Reply:
fixed